### DISCUSSION TITLE
How to implement data fetchers for nested mutation.

### DISCUSSION THREAD

**Discussion by sriramgopalan-awx** [MichalLytek/type-graphql#1427] [Q&A]
```
type Mutation {
    topLevel: MyTopLevelMutation!
}

type MyTopLevelMutation {
 
    signUp(input: UserInput!): UserPayload!
    
    #...more mutations
}

type UserInput {
   first: String!
   last: String!
}

type UserPayload {
    success: Boolean!
    id: ID!
}
```

As you see in the above schema, I have a nested mutation structure and the operations are not at the root level. Now how do I implement data fetchers for `topLevel` field  and for `sign up`? Any help will be really appreciated. 
 
 https://github.com/Netflix/dgs-framework/discussions/528 - In this issue they refer to testing but I would like to know the implementation of the data fetchers.

**Comment by MichalLytek** [comment]
Nested mutations are an invalid pattern - see #1219 #1045 #781 #712 #312 #64
